src/components/alt-button/FavoriteButton.module.css

```diff
@@ -1,46 +1,71 @@
 @keyframes scaleX {
-    0% {
-        transform: scale(1, 1);
```

Comment:
Does this mean something is wrong with my Prettier/Eslint setup?

Reply:
I think you might need this in your VS Code settings?
```json
{
  "editor.tabSize": 2
}
```

Reply:
er, actually read the diff backwards. I know @dylanjeffers just added stylelint - what's the rule here with indents?

Reply:
hey hey! so i have this in my .vscode/settings.json for the project:

Reply:
but actually i think this means everything is working for you, im surprised that file wasnt caught by stylelint already though...

Reply:
oh actually... looks like its using tabs not spaces for stylelint, which i think we don't want to do... ill make a quick change!

Reply:
Also I think it is behind master by 8 commits, which means it's missing your stylelint change. :)

Reply:
yeah maybe we rebase with master first and see where that takes us?

Reply:
ohh gotcha thanks ill try that and see what happens

Reply:
looks like that fixed it ☺️